airbyte-integrations/connectors/source-stripe/setup.py

```diff
@@ -25,7 +25,7 @@
 
 from setuptools import find_packages, setup
 
-MAIN_REQUIREMENTS = ["airbyte-cdk", "stripe"]
+MAIN_REQUIREMENTS = ["airbyte-cdk~=0.1", "stripe"]
```

Comment:
Could you specify the version of stripe lib too?

Reply:
Done.